src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/proxy.py

```diff
@@ -34,12 +28,76 @@ def set_http_proxy(host, port):
     if not port:
         raise ValueError('Missing port')
 
-    os_platform = platform.system()
+    __get_proxy_instance().set_http_proxy(host, port)
 
+def __get_proxy_instance():
```

Comment:
Methods that start with `__` have a special meaning in Python.

"Generally, double leading underscores should be used only to avoid name conflicts with attributes in classes designed to be subclassed."
from: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables